features/event-publisher/org.wso2.carbon.event.output.adapter.server.feature/src/main/resources/conf_templates/templates/repository/conf/output-event-adapters.xml.j2

```diff
@@ -98,6 +98,8 @@
         <property key="mail.smtp.port">{{output_adapter.email.port}}</property>
         <property key="mail.smtp.starttls.enable">{{output_adapter.email.enable_start_tls}}</property>
         <property key="mail.smtp.auth">{{output_adapter.email.enable_authentication}}</property>
+        <property key="mail.smtp.signature">{{output_adapter.email.username}}</property>
```

Comment:
Change key to {{output_adapter.email.signature}}

Reply:
as per the default configuration signature should be the username it self

Reply:
changed